spec/std/file_spec.cr

```diff
@@ -711,7 +742,7 @@ describe "File" do
       with_tempfile("append-existing.txt") do |path|
         File.write(path, "hello")
         File.read(path).should eq("hello")
-        File.write(path, " world", mode: "a")
+        File.write(path, " world", mode: File::Mode.flags(Create, Append))
```

Comment:
Can we have a simpler way to use `File.write`? Maybe this imply moving arguments from it to named arguments to match the `File.open` ones.

Reply:
Using `File.write` with a custom mode is very rare. I think optimizing this would be a little premature.